### PR TITLE
DO NOT MERGE: docs(rxFeedback): FRMW-175-Migrate rxFeedback to API docs

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-175

- [x] Dev LGTM
- [x] Dev LGTM

Migrate `rxFeedback` to API Documentation.

### CODE REVIEW

**Comment by bry4n** [comment]
LGTM

**Comment by CITguy** [comment]
DO NOT MERGE: waiting for changes in #1226

File: src/rxFeedback/rxFeedback.js [hunk 1]
```diff
@@ -118,7 +127,29 @@ angular.module('encore.ui.rxFeedback', ['ngResource'])
  * @name rxFeedback.directive:rxFeedback
  * @restrict E
  * @scope
- * @description [TBD]
+ * @description
+ * ## Custom Submission Function
+ * 
+ * The `rxFeedback` directive allows you to define an `on-submit` attribute that points to a custom function for the
+ * purposes of overriding the default submission logic.  This function should accept a *single argument* for a
```
Comment (TheNando) [inline src/rxFeedback/rxFeedback.js]: Can you change "should accept" to read "accepts"?

Comment (100stacks) [inline src/rxFeedback/rxFeedback.js]: Okay will do :+1:

**Comment by TheNando** [comment]
LGTM

**Comment by CITguy** [comment]
PR was not ready to merge. Documentation was lost for rxFeedback.

**Comment by CITguy** [comment]
I'll submit a new PR to put the documentation where it needs to be.

**Pull request merged**